src/chrome/content/rules/rns.online.xml

```diff
@@ -0,0 +1,12 @@
+<ruleset name="rns.online">
+	<target host="rns.online" />
+	<target host="www.rns.online" />
+	<target host="cdn.rns.online" />
+	<test url="http://cdn.rns.online" />
+	<test url="http://cdn.rns.online/" />
+	<test url="http://cdn.rns.online?" />
+	<test url="http://cdn.rns.online#" />
+	<test url="http://cdn.rns.online/?" />
+	<exclusion pattern="^http://cdn\.rns\.online/?(?:$|\?|#)" />
```

Comment:
A trailing slash is required.
Please also avoid non-capturing groups, as our code style recommends.